test/torch/pointers/test_pointer_tensor.py

```diff
@@ -481,7 +481,7 @@ def test_share_get(workers):
     tensor = torch.tensor([1, 2, 3])
     ptr = tensor.send(bob)
 
-    ptr = ptr.share()
+    ptr = ptr.share(crypto_provider=bob)
```

Comment:
that's weird, can you declare alice, bob and secure_worker (from the dict workers)
and do 
`ptr.share(bob, alice, crypto_provider=secure_worker)` ?